packages/suite/src/components/wallet/TokenIconSetWrapper.tsx

```diff
@@ -1,63 +1,60 @@
-import { selectCoinDefinitions } from '@suite-common/token-definitions';
+import { EnhancedTokenInfo } from '@suite-common/token-definitions';
 import { NetworkSymbol } from '@suite-common/wallet-config';
-import { selectCurrentFiatRates, selectLocalCurrency } from '@suite-common/wallet-core';
-import { Account } from '@suite-common/wallet-types';
 import { TokenIconSet } from '@trezor/product-components';
 import { BigNumber } from '@trezor/utils';
 
-import { useSelector } from 'src/hooks/suite';
-import {
-    TokensWithRates,
-    enhanceTokensWithRates,
-    getTokens,
-    sortTokensWithRates,
-} from 'src/utils/wallet/tokenUtils';
+import { sortTokensWithRates } from 'src/utils/wallet/tokenUtils';
 
 type TokenIconSetWrapperProps = {
-    accounts: Account[];
+    expanded?: boolean;
```

Comment:
this should be togglable, meaning, you could turn off feature that expansion is possible